devices/esp32.rst

```diff
@@ -52,6 +52,17 @@ Some notes about the pins on the ESP32:
       - platform: gpio
         name: "Pin GPIO23"
         pin: GPIO23
+        
+** Note **
+
+Espp32 devices can sometime come without any flash layout. Just use esphome-flasher 117 to flash you bin for the first time via serial - it takes care of partitioning the esp32s.
```

Comment:
1. Typo in espp32
2. The syntax in RST for a note is
```
.. note::

    ESP32 devices ...
```
3. Please direct the users to the esphome dashboard or https://web.esphome.io instead - those are more user-friendly for flashing devices

Reply:
I tried [](https://web.esphome.io/) first and it did not sort out the flash layout. So maybe this is a bug with the web flasher?

Reply:
This is what led to the flasher tool which did sort the flash layout as per comment in the title

Reply:
The issue there is you probably tried flashing the wrong binary file, which is only the application, not the combined binary that contains all the partitions.